constance/admin.py

```diff
@@ -272,11 +272,15 @@ def changelist_view(self, request, extra_context=None):
                         config_values.append(
                             self.get_config_value(name, options, form, initial)
                         )
-
-                context['fieldsets'].append({
+                fieldset_context = {
                     'title': fieldset_title,
                     'config_values': config_values
-                })
+                }
+
+                if (settings.CONFIG_COLLPASED_FIELDSETS and
+                        fieldset_title in settings.CONFIG_COLLPASED_FIELDSETS):
+                    fieldset_context['collapse'] = True
+                context['fieldsets'].append(fieldset_context)
```

Comment:
This shouldn't use yet another setting, just make the `CONSTANCE_CONFIG_FIELDSETS` have an optional second format using a dict:
```python
CONSTANCE_CONFIG_FIELDSETS = {
    'General Options': ('SITE_NAME', 'SITE_DESCRIPTION'),
    'Theme Options': ('THEME',),
}
```
and this is the same but with a collapsible form:
```python

CONSTANCE_CONFIG_FIELDSETS = {
    'General Options': {
        'classes': 'collapse',
        'fields':('SITE_NAME', 'SITE_DESCRIPTION'),
    },
    'Theme Options': ('THEME',),
}
```

EDIT: actually, let's follow Django's way for specifying fieldsets: https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fieldsets